.rubocop.yml

```diff
@@ -0,0 +1,5 @@
+inherit_from:
+  - https://raw.githubusercontent.com/cookpad/guides/master/.rubocop.yml
+
+Rails:
```

Comment:
This definitely shouldn't be here. This code base is a Ruby gem, and is compeltely unrelated with Rails. We shouldn't event have to `disable` rails rules.

Reply:
Can be removed after removing cookpad's guidelines.